components/director/internal/domain/runtime/service_test.go

```diff
@@ -58,6 +58,31 @@ func TestService_Create(t *testing.T) {
 			Input:       modelInput,
 			ExpectedErr: nil,
 		},
+		{
+			Name: "Returns error when name is empty",
```

Comment:
you test the same thing twice: in model package and here. Here you should have a test that checks if an input is validated - one scenario will be enough.